tests/php/NetworkSiteConnectionsTest.php

```diff
@@ -272,7 +323,7 @@ public function test_pull() {
 		\WP_Mock::userFunction(
 			'update_post_meta', [
 				'times'  => 1,
-				'args'   => [ \WP_Mock\Functions::type( 'int' ), 'dt_original_post_id', 2 ],
+				'args'   => [ \WP_Mock\Functions::type( 'int' ), 'dt_original_post_id', 111 ],
```

Comment:
I can not figure out why this was passing. The `get_post()` mock above is using `111` so that's what should have been passed.